### PR TITLE
Clarify `new WebSocket()` API

### PR DESCRIPTION
**Pull request by cacheburner**
This constructor has been functional for a couple months, so we should update the docs.

### CODE REVIEW

File: content/workers/learning/using-websockets.md [hunk 1]
```diff
@@ -173,6 +173,10 @@ async function websocket(url) {
 }
 ```
 
+## WebSocket Compression
+
+Cloudflare Workers does not currently support WebSocket compression, and so unlike browsers, using `new WebSocket(url)` in a Worker will not set `Sec-WebSocket-Extensions: permessage-deflate`.
```
Comment (mattdeboard) [inline content/workers/learning/using-websockets.md]: Copy-editing suggestion only, here:

> Cloudflare Workers does not currently support WebSocket compression. This means that, unlike browsers, using `new WebSocket(url)` in a Worker will not set `Sec-WebSocket-Extensions: permessage-deflate`.

Comment (cacheburner) [reply content/workers/learning/using-websockets.md]: https://github.com/cloudflare/cloudflare-docs/pull/6703/commits/65f41f47bc5e8e41463db30e68518b6456807b4c

File: content/workers/learning/using-websockets.md [hunk 2]
```diff
@@ -173,6 +173,10 @@ async function websocket(url) {
 }
 ```
 
+## WebSocket Compression
```
Comment (deadlypants1973) [inline content/workers/learning/using-websockets.md]:
```suggestion
## WebSocket compression
```

File: content/workers/learning/using-websockets.md [hunk 3]
```diff
@@ -139,9 +139,9 @@ For an example of this in practice, refer to the [`websocket-template`](https://
 
 ## Writing a WebSocket client
 
-A Worker can also establish a WebSocket connection to a remote server.
+Cloudflare Workers supports the `new WebSocket(url)` constructor, and so a Worker can establish a WebSocket connection to a remote server in the same manner as the client implementation described above.
```
Comment (deadlypants1973) [inline content/workers/learning/using-websockets.md]:
```suggestion
Cloudflare Workers supports the `new WebSocket(url)` constructor. A Worker can establish a WebSocket connection to a remote server in the same manner as the client implementation described above.
```

File: content/workers/learning/using-websockets.md [hunk 4]
```diff
@@ -173,6 +173,10 @@ async function websocket(url) {
 }
 ```
 
+## WebSocket compression
+
+Cloudflare Workers does not currently support WebSocket compression. This means that, unlike browsers, using `new WebSocket(url)` in a Worker will not set `Sec-WebSocket-Extensions: permessage-deflate`.
```
Comment (deadlypants1973) [inline content/workers/learning/using-websockets.md]:
```suggestion
Cloudflare Workers does not currently support WebSocket compression. This means that, unlike browsers, using `new WebSocket(url)` in a Worker will not set the `Sec-WebSocket-Extensions: permessage-deflate` header.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**